webextension/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "WebExtension-shield-study",
+  "applications": {
+    "gecko": { "id": "webextensionExperiment@shield.mozilla.org" }
+  },
+  "version": "1.0.0",
+  "manifest_version": 2,
+  "sidebar_action": {
+    "default_title": "Demonstration Sidebar",
+    "default_panel": "sidebar.html"
+  },
+  "experiment_apis": {
+    "prefs": {
+      "schema": "./privileged/prefs/schema.json",
+      "parent": {
+        "scopes": ["addon_parent"],
+        "script": "./privileged/prefs/prefs.jsm",
+        "paths": [["hello"]]
```

Comment:
This should presumably be `"prefs"` instead of `"hello"`.  This line will cause your API implementation to be loaded when an extension references `browser.hello` but of course you want it to be loaded when an etension references `browser.prefs`